### PR TITLE
Feat/bulkupload

### PR DESCRIPTION
**Pull request by cnlangzi**
### Changes
- added `BulkUpload`, `GetLookupHash` in winsdk

### Fixes
-

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- blobber:
- 0chain:
- system_test:
- zboxcli: https://github.com/0chain/zboxcli/pull/496
- zwalletcli:
- Other: ...

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  |   |
| gosdk  | feat/bulkupload  |
| zboxcli  |   |
| zwalletcli  |   |
| 0chain  |   |
| blobber  |   |
| authorizer  |   |
| 0box  |   |
| 0dns  |   |

**Comment by service-0chain** [comment]
Manual system tests [cancelled] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  |   |
| gosdk  | feat/bulkupload  |
| zboxcli  |   |
| zwalletcli  |   |
| 0chain  |   |
| blobber  |   |
| authorizer  |   |
| 0box  |   |
| 0dns  |   |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | false  |
| system-tests  |   |
| gosdk  | feat/bulkupload  |
| zboxcli  |   |
| zwalletcli  |   |
| 0chain  |   |
| blobber  |   |
| authorizer  |   |
| 0box  |   |
| 0dns  |   |

File: mobilesdk/zbox/storage.go [hunk 1]
```diff
@@ -334,20 +335,23 @@ func MultiUpload(allocationID string, workdir string, jsonMultiUploadOptions str
...
+	isUpdates := make([]bool, totalUploads)
 	for idx, option := range options {
 		filePaths[idx] = option.FilePath
 		fileNames[idx] = option.FileName
 		thumbnailPaths[idx] = option.ThumbnailPath
 		remotePaths[idx] = option.RemotePath
 		chunkNumbers[idx] = option.ChunkNumber
+		isUpdates[idx] = false
 		encrypts[idx] = option.Encrypt
+    isUpdates[idx] = false
```
Comment (peterlimg) [inline mobilesdk/zbox/storage.go]: This `isUpdates[idx] = false` is set twice, see line 345 above

Comment (cnlangzi) [reply mobilesdk/zbox/storage.go]: fixed

**Review by din-mukhammed — APPROVED** [review]

**Review by guruhubb — APPROVED** [review]

**Pull request merged**